org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/TelemetryManager.java

```diff
@@ -136,6 +171,11 @@ public void onBuildFinished(long buildFinishedTime) {
 		telemetryEvent(JAVA_PROJECT_BUILD, properties);
 	}
 
+	private boolean isUnresolvedImportError(IMarker marker) {
```

Comment:
Could be moved into `ResourceUtils` with the other marker helper methods.

Reply:
Nice.